client/modules/food/components/packing/Packages.js

```diff
@@ -84,9 +109,7 @@ class Packages extends Component {
             <TableHeaderColumn dataField="status" >
               Status
             </TableHeaderColumn>
-            <TableHeaderColumn
-              dataFormat={this.getActionButtons}
-            />
+            <TableHeaderColumn dataFormat={this.getActionButtons} />
```

Comment:
The unpack and delivered buttons are not fitting on my browser either.  Can you change this line to `<TableHeaderColumn dataFormat={this.getActionButtons} width="200px"/>`?  That should make the column big enough so they fit side by side.